src/cloud/env/variables-build.md

```diff
@@ -86,6 +86,21 @@ stage:
     SCD_COMPRESSION_TIMEOUT: 800
 ```
 
+### `SCD_NO_PARENT`
+
+-  **Default**—`false`
+-  **Version**—Magento 2.3.0 and later
```

Comment:
Here should be 2.4.2 and later, firstly it was 2.3.0 but it was fixed in the code of ece-tools